torch/_torch_docs.py

```diff
@@ -5810,6 +5810,51 @@ def merge_dicts(*dicts):
     tensor([  435.,  1335.,  2235.,  3135.])
 """.format(**multi_dim_common))
 
+add_docstr(torch.nansum,
+           r"""
+nansum(input, dtype=None) -> Tensor
+
+Return the sum of array elements over a given axis treating Not a Numbers (NaNs) as zero.
+
+Args:
+    {input}
+    {dtype}
```

Comment:
dtype is an optional arg

Reply:
Updated. 

Note: `torch.sum` also has the same structure. Should be updated.